source/main.css

```diff
@@ -0,0 +1,126 @@
+.info {
+    margin: 0px auto;
```

Comment:
When a CSS value goes to 0 you can just drop the unit (in this case `px`). So just: `margin: 0 auto;`.